README.md

```diff
@@ -3,4 +3,11 @@
 Generates a Selenium Automation script (.side) for adding attendees to a Bevy Event.
 
 Tested on:
-1. Chrome
\ No newline at end of file
+1. Chrome
```

Comment:
Need more info here. Could you please move the README changes to a separate PR so that we can work on it in length?